docs/xdr/usecases/playbook/whoIs.md

```diff
@@ -0,0 +1,35 @@
+# WhoIs module configuration
+
+This use case describes how to use WhoIs module to retrieve information.
+
+## Prerequisites
+
+- A SEKOIA.IO XDR licence
+- The user that configures the Playbook should have a role that contains:
+	* The SYMPHONY permissions
+- An API Key with a role that contains at least the following permissions:
+	* `SIC_READ_ALERTS`
+	* `EVENTS_READ_STATS`
+	* `EVENTS_READ_DATASETS`
+	* `SIC_READ_EVENT_STATS`
+
+!!! note
+    To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).
+
+## Playbook configuration
+
+You can find the configuration below: 
+
+![Playbook WhoIS](docs/assets/playbooks/library/UseCases/WhoIS.png)
```

Comment:
The image is not displayed in the preview, is it normal?

Reply:
image is available [here](https://github.com/SEKOIA-IO/documentation/tree/main/docs/assets/playbooks/library/UseCases)
As this page is not yet linked in the documentation tree, it is not displayed

It will be displayed when this PR will be validated and that I link this page to the documentation